src/cargo/core/features.rs

```diff
@@ -957,7 +962,10 @@ impl CliUnstable {
             "multitarget" => stabilized_warn(k, "1.64", STABILISED_MULTITARGET),
             "rustdoc-map" => self.rustdoc_map = parse_empty(k, v)?,
             "terminal-width" => self.terminal_width = Some(parse_usize_opt(v)?),
-            "sparse-registry" => self.sparse_registry = parse_empty(k, v)?,
+            "sparse-registry" => {
+                stabilized_warn(k, "1.66", STABILISED_SPARSE_REGISTRY);
+                self.sparse_registry = parse_empty(k, v)?;
```

Comment:
It's great to see we have this as warning! Could we have a note here for developers telling that when should we remove `sparse_registry` remove `CliUnstable`?

(Like, after sparse protocol becomes the default and we can remove it and turn it to a hard error)